style.css

```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap');
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.hero-section {
+  width: 95vw;
+  height: 95vh;
+}
+
+.nav-bar {
+  height: 8%;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  height: 90%;
+  align-items: start;
+  justify-content: center;
+  background-image: url("./assets/images/header-shapes-mobile.svg");
+  background-size: 100%;
```

Comment:
Kindly make sure the `margin` and `padding` are met with the one provided on the Figma design.
As you can see your design
![Screenshot from 2022-09-21 18-14-54](https://user-images.githubusercontent.com/97100091/191558211-04fcd7cf-b309-49ac-94c3-7cdd3b6d38f1.png)

comparing to the origin design
![Screenshot from 2022-09-21 18-21-48](https://user-images.githubusercontent.com/97100091/191558472-0fc7ee51-a2e4-45bb-8aec-26e5d98e7239.png)